Bottom: d6042f47e890e2e27465e6eddcf69b8e0e9f7695
Top:    6c4d00650d47cbb6cfaadffe733bf35821e043c3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 15:08:15 +0000

Refresh of attempt-to-refactor-the-step

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 446739d..df42091 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -169,6 +169,7 @@ PCB_SRCS = \
 	hid/common/object3d.h \
 	hid/common/quad.c \
 	hid/common/quad.h \
+	hid/common/step_id.h \
 	hid/common/vertex3d.c \
 	hid/common/vertex3d.h \
 	hid/hidint.h 
diff --git a/src/hid/common/contour3d.c b/src/hid/common/contour3d.c
index a9976cc..13e865e 100644
--- a/src/hid/common/contour3d.c
+++ b/src/hid/common/contour3d.c
@@ -2,7 +2,7 @@
 #include <stdbool.h>
 #include <stdio.h>
 
-#include "step.h"
+#include "step_id.h"
 #include "quad.h"
 #include "contour3d.h"
 
diff --git a/src/hid/common/edge3d.c b/src/hid/common/edge3d.c
index 2d34558..c67fa95 100644
--- a/src/hid/common/edge3d.c
+++ b/src/hid/common/edge3d.c
@@ -3,7 +3,7 @@
 #include <stdio.h>
 #include <glib.h>
 
-#include "step.h"
+#include "step_id.h"
 #include "edge3d.h"
 
 edge_info *
diff --git a/src/hid/common/face3d.c b/src/hid/common/face3d.c
index 8e8f8c0..0f9f327 100644
--- a/src/hid/common/face3d.c
+++ b/src/hid/common/face3d.c
@@ -2,7 +2,7 @@
 #include <stdbool.h>
 #include <stdio.h>
 
-#include "step.h"
+#include "step_id.h"
 #include "quad.h"
 #include "contour3d.h"
 #include "appearance.h"
diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index a79d255..273113c 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -5,7 +5,7 @@
 
 #include <glib.h>
 
-#include "step.h"
+#include "step_id.h"
 #include "quad.h"
 #include "vertex3d.h"
 #include "contour3d.h"
diff --git a/src/hid/common/vertex3d.c b/src/hid/common/vertex3d.c
index 8e584ab..8abef8f 100644
--- a/src/hid/common/vertex3d.c
+++ b/src/hid/common/vertex3d.c
@@ -3,7 +3,7 @@
 #include <stdbool.h>
 #include <stdio.h>
 
-#include "step.h"
+//#include "step.h"
 #include "vertex3d.h"
 
 static int global_vertex3d_count;
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index f7480a3..491154b 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -15,7 +15,7 @@
 #include "gui-pinout-preview.h"
 #include "pcb-printf.h"
 
-#include "hid/common/step.h"
+#include "step.h"
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
 #include "hid/common/contour3d.h"
